pkg/storaged/format-dialog.jsx

```diff
@@ -61,6 +67,7 @@ export function initial_tab_options(client, block, for_fstab) {
     const options = { };
 
     utils.get_parent_blocks(client, block.path).forEach(p => {
+        options.nofail = true;
```

Comment:
Should we have a comment why we always add `nofail` because Cockpit always deals with non-primary disks?

Reply:
Good idea, I'll do that.

Reply:
Yes, should be a comment in the code :)